tensorflow/stream_executor/cuda/cuda_rng.cc

```diff
@@ -15,8 +15,9 @@ limitations under the License.
 
 #include "tensorflow/stream_executor/cuda/cuda_rng.h"
 
+#if !defined(PLATFORM_WINDOWS)
 #include <dlfcn.h>
-
+#endif
```

Comment:
I think this is a stray `#include` that is no longer used, and we can just delete it.